dist/externalBank.js

```diff
@@ -151,6 +151,10 @@ const internalErrors = {
   INVALID_BLOCK_MONEY: {
     code: 'external-37',
     message: 'Invalid block number'
+  },
+  BLOCKED_CLIENT_ACCOUNT: {
+    code: 'external-38',
+    message: 'The customer`s account is blocked by the issuer'
```

Comment:
no es mejor `The customer's account is blocked by the issuer`? no se porque me genera temor tener un ` en medio de una cadena de string, tipo que algun micro o el front end tenga fallos por tener ese simbolo en medio del string (capaz y es paranoia mia)